OrbitService/CaptureServiceImpl.cpp

```diff
@@ -91,30 +91,52 @@ class GrpcCaptureEventSender final : public CaptureEventSender {
     CHECK(reader_writer_ != nullptr);
   }
 
+  ~GrpcCaptureEventSender() {
```

Comment:
Could you add `override` to this (clang-tidy)?
```suggestion
  ~GrpcCaptureEventSender() override {
```

Reply:
Done.